docs/HowTo/Configure/Consensus-Protocols/Add-Validators.md

```diff
@@ -419,3 +423,13 @@ To add or remove validators without voting:
 !!! caution
     Do not specify a transition block in the past. Specifying a transition block in the past could
     result in unexpected behaviour, such as causing the network to fork.
+
+### Adding and removing validators using a smart contract
+
+You can deploy the validator smart contract in a QBFT network by specifying the contract details in the
+[genesis file](QBFT.md#genesis-file).
```

Comment:
It might be worth mentioning here that using the genesis file will only work when you're pre-deploying the smart contract, i.e. at the start of the network.
For an existing network, you would need to compile and deploy the contract using a transaction, obtain the contract address from the receipt and use that in a transition.

Reply:
Good point, thanks! I updated this.